backend-api/infra/queue/vendorProcessing.yaml

```
@@ -0,0 +1,49 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Resources:
+  VendorProcessingSQS:
+    Type: AWS::SQS::Queue
+    Properties:
+      MessageRetentionPeriod: 1209600
+      VisibilityTimeout: 10
+      DelaySeconds: 5
+      ReceiveMessageWaitTimeSeconds: 5
+      KmsMasterKeyId: !Ref VendorProcessingKeyAlias
+      RedrivePolicy:
+        deadLetterTargetArn: !GetAtt VendorProcessingDLQ.Arn
+        maxReceiveCount: 360
+
+  VendorProcessingDLQ:
+    Type: AWS::SQS::Queue
+    Properties:
+      MessageRetentionPeriod: 1209600
+      KmsMasterKeyId: !Sub VendorProcessingKMSEncryptionKey
+
+  VendorProcessingKMSEncryptionKey:
+    Type: AWS::KMS::Key
+    Properties:
+      Description: A KMS Key for encrypting the VendorProcessing SQS
```

question/preference check: should I have a space here?

```suggestion
      Description: A KMS Key for encrypting the Vendor Processing SQS
```

---

Think I prefer the space, going to change

---

Updated